src/nemo_safe_synthesizer/generation/vllm_observability.py

```
@@ -0,0 +1,551 @@
+        return {}
+    out: dict[str, Any] = {}
+    for spec in _PROBE_FIELDS:
+        try:
+            section = getattr(vcfg, spec.section, None)
+            if section is None:
+                continue
+            value = _first_present(section, spec.sources)
+            if value is not None:
+                out[spec.out_key] = spec.transform(value)
+        except Exception:  # noqa: BLE001 — degrade one field, not the whole probe
+            continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)